telluric/constants.py

```diff
@@ -20,3 +20,10 @@
 DEFAULT_SRID = WGS84_SRID
 #: Default CRS, set to :py:data:`~telluric.constants.WGS84_CRS`.
 DEFAULT_CRS = WGS84_CRS
+
+
+def _mercator_zoom_to_resolution(zoom_level):
+    return (2 * 20037508.342789244) / (256 * pow(2, zoom_level))
+
+
+mercator_zoom_to_resolution = dict((i, _mercator_zoom_to_resolution(i)) for i in range(19))
```

Comment:
Minor nitpick: can this be `MERCATOR_ZOOM_TO_RESOLUTION` or `MERCATOR_RESOLUTION_MAPPING`?